runtimes/pythonrt/py.go

```diff
@@ -63,7 +63,27 @@ func parsePyVersion(s string) (major, minor int, err error) {
 	return
 }
 
+func isFile(path string) bool {
+	info, err := os.Stat(path)
+	if err != nil {
+		return false
+	}
+
+	return !info.IsDir()
+}
+
+const exeEnvKey = "AK_PYTHON"
+
 func findPython() (string, error) {
+	exePath := os.Getenv(exeEnvKey)
+	if exePath != "" {
+		if !isFile(exePath) {
```

Comment:
any chance it is a file but not executable? 
probably an overkill check but in limited environments (like docker) we might block users from doing stuff (the docker user, not any user) and it might be helpful to know at first hand

Reply:
When we detect the python we call `python --version` to get the version, it'll fail there.